frontend/packages/dev-console/integration-tests/features/helm/helm-Installation-View.feature

```diff
@@ -0,0 +1,37 @@
+Feature: Helm Chart Installation View
+    As a user, I should be able switch between YAML and Form view to install Helm Chart
+
+
+Background:
+    Given user logged into the openshift application
+    And user is at developer perspecitve
+
+
+@regression
+Scenario: Grouping of Helm multiple chart versions together in developer catalog
+    Given user is at the developer catalog page
+    When user checks the Helm Charts checkbox
+    And user searches for the 'Node-ex-k' helm chart
```

Comment:
Data updation is required 
Nodejs Ex K v0.2.1

Reply:
Updated